ArjunaCore/arjuna/classes/com/arjuna/ats/arjuna/tools/osb/mbean/UidWrapper.java

```diff
@@ -53,11 +54,13 @@ public static String getRecordWrapperTypeName() {
 
 	private String name;
 
-	private ObjStoreBrowser browser;
-	private String beantype;
-    private String className;
-	private String ostype;
-	private Uid uid;
+	private final ObjStoreBrowser browser;
+	private final String browserMBeanName;
+	private final String beantype;
+    private final String className;
```

Comment:
misaligned indent

Reply:
You have replaced spaces with tabs in this file (I thought our style checker detected these issues?).

Reply:
I did nothing with indentation.  My point was to add the `final` identifier.
The trouble is that the file itself was wrongly indented before I did the changes here. Some lines were with space (the one with `className`) the others were with tabs.

Our style checker was permitted to be run only on `rts` and `cdi` modules. I got no permission to run it at other places.

What should I do here? Should I change all changed lines to have tab indentation? Or spaces?

Reply:
If the tab was already present then no problem.